user-journeys/application_developer-foundational.html

```diff
@@ -16,80 +16,112 @@
+<br><br>
+        <a href="https://kubernetes.io/docs/tutorials/kubernetes-basics/explore-intro/">Viewing Pods and Nodes</a>
+<br><br>
+        Install a sample application. Recommendations? 
       </div>
 
       <div class="topheader" id="2">
          <span class="numberCircle"><span>2</span></span></i>&nbsp;&nbsp;Deploy, scale, and update an application
       </div>
 
       <div class="docsection1">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id felis non enim faucibus lacinia. Aliquam massa mauris, interdum a ex ut, sagittis rutrum nulla. In pellentesque est at molestie fringilla. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec eu mi at velit lacinia venenatis ac nec sem. In volutpat pellentesque dui ut commodo. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Duis sollicitudin eleifend felis non facilisis. Pellentesque leo urna, congue id auctor non, varius a nunc. Duis ultrices, odio ut hendrerit suscipit, nisi mauris dignissim mauris, nec bibendum ante neque ut augue. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam condimentum libero sit amet rutrum fermentum. Proin posuere condimentum odio. Aenean volutpat, ex vel tincidunt tincidunt, massa odio condimentum lectus, vel iaculis libero dolor in nisi.
+        To deploy and manage an application in Kubernetes, you write a manifest, also called a configuration or config file. The manifest is written in YAML or JSON, and describes the desired state of the application and related resources as Kubernetes should maintain them. (if this content stays, needs to be moved)
 <br><br>
-        Curabitur bibendum tempor mi, vel lacinia nisi vulputate ac. Nulla dignissim consectetur nisl nec tincidunt. Etiam pharetra facilisis sapien, non gravida velit fermentum sed. Ut ac ultrices nunc, in vestibulum urna. Suspendisse accumsan euismod felis, sit amet rhoncus neque volutpat luctus. Aliquam tincidunt pellentesque mauris, sed tempus diam. Mauris in elit eget justo tempor suscipit.
+        TUTORIAL? combine:
 <br><br>
-        Aenean suscipit arcu ac leo tincidunt tempus. Donec maximus tellus libero, ac ullamcorper magna lobortis ac. Integer mollis nisl vitae magna gravida, nec ornare ex consectetur. Sed mattis tincidunt nisi, at consequat tellus malesuada non. Integer vel semper nisi, ut fringilla velit. Nam felis ex, congue non dui vitae, sollicitudin convallis turpis. Phasellus porttitor maximus turpis, in varius nibh fermentum aliquam. Cras finibus lacus non diam porttitor porttitor. Nulla fringilla sagittis nibh nec condimentum. Duis egestas mauris nec dolor hendrerit ullamcorper sit amet in mi. Phasellus sollicitudin justo diam.
+        <a href="https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-intro/">Using kubectl to Create a Deployment</a>" -- this page is primarily conceptual/expanatory material.
```

Comment:
I think we're more likely to find what we need under Tasks, not so much under Tutorials.

Here's a Deployment task. (Ignore the fact that it is currently busted unless you have a Kubernetes 1.8 cluster.)

[Run a Stateless App Using a Deployment](https://kubernetes.io/docs/tasks/run-application/run-stateless-application-deployment/)